docs/releases/1.10.txt

```diff
@@ -33,6 +33,17 @@ search engine. You can search across multiple fields in your relational
 database, combine the searches with other lookups, use different language
 configurations and weightings, and rank the results by relevance.
 
+New-style middleware
+--------------------
+
+:doc:`A new style of middleware is introduced </topics/http/middleware>` to
+solve the lack of strict request/response layering of the old-style of
+middleware described in `DEP 0005
+<https://github.com/django/deps/blob/master/draft/0005-improved-middleware.rst>`_.
```

Comment:
Yes, I'm going to look over this tomorrow, bring the DEP up to date, and submit it to the technical board for a vote.